microcosm_flask/runserver.py

```diff
@@ -12,6 +12,7 @@ def parse_args(graph):
     default_enable_profiling = graph.config.flask.enable_profiling
 
     parser = ArgumentParser()
+    parser.add_argument("-s", default="127.0.0.1")
```

Comment:
What's this for?

Reply:
leftover from manual testing - will remove